feat: [CDS-59301]: support for infrastructure tags (#1711)
harness/developer-hub: docs/continuous-delivery/get-started/services-and-environments-overview.md

![](./static/services-and-environments-overview-14.png)

#### Infrastructure Tags

> Tags can be attached to Infrastructure definitions representing their characteristics. These tags can be key value pairs. 

Reviewer: ```suggestion
Tags can be attached to infrastructure definitions representing their characteristics. These tags can be key value pairs. 
```

![](./static/services-and-environments-overview-28.png)

> When you select the **Infrastructure Definition** to use for that stage, the attached tags can be accessed using their keys with the expression ```<+infra.tags.tag_key>```. This expression is available for use throughout the stage. 

Reviewer: ```suggestion
When you select the **Infrastructure Definition** to use for a stage, the attached tags can be accessed using their keys with the expression, ```<+infra.tags.tag_key>```. This expression is available for use throughout the stage. 
```

> An example use case would be to skip certain steps in pipeline based on the tags attached to the infrastructure. 

Reviewer: ```suggestion
For example, skipping certain steps in pipeline based on the tags attached to the infrastructure. 
```
